code/style.css

```diff
@@ -1,4 +1,150 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing:border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  background-color: whitesmoke;
 }
+
+.header {
+  width: 100%;
+  height: 100px;
+  background-color: whitesmoke;
+  padding: 30px 30px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 40px;
+}
+
+.logo {
+  line-height: 0.7;
+}
+
+.logo h1 {
+  font-size: 45px;
+}
+
+.news {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 40px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+p {
+  margin: 0 10px;
+  font-size: 14px;
+}
+
+h2 {
+  margin: 0 10px;
+  font-size: 27px;
+}
+
+h3 {
+  margin: 0 10px;
+  font-size: 20px;
+}
+
+.headnews img {
+  width: 100%;
+  height: auto;
+}
+
+.news2 img {
+  width: 100%;
+  height: auto;
+}
+
+.news3 img {
+  width: 100%;
+  height: auto;
+}
+
+.news4 img {
+  width: 100%;
+  height: auto;
+}
+
+.news5 img {
+  width: 100%;
+  height: auto;
+}
+
+
+/*for mobile*/
+@media (max-width: 767px) {
+  .navbar {
+    display: none;
+  }
+}
+
+/*for tablet*/
+@media (min-width: 768px) {
+  .navbar {
+    display:flex;
+    flex-direction: row;
+   }
+   
+  .navbar ul li {
+    display: inline;
+   }
+   
+  .navbar ul li a {
+    color: black;
+    font-size: 15px;
+    text-transform: uppercase;
+    padding: 5px;
+   }
+   
+  .navbar ul li a:hover {
+    background-color: #999;
+    border-radius: 10%;
+   }
+
+  .news {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    row-gap: 30px;
+    column-gap: 10px;
+   }
+
+  .logo {
+    line-height: 0.7;
+   }
+  
+  .logo h1 {
+    font-size: 65px;
+   }
+
+   .headnews {
+    grid-area: 1 / 1 / 2 / 3;
+   }
+} 
+
+/*for desktop*/
+@media (min-width: 1024px) {
+ 
+  .news {
+    display: grid;
+    grid-template-columns: 25% 25% 25% 25%;
```

Comment:
Could be a good tip to use repeatfunction here, like: grid-template-columns: repeat(4, 25%)
Saves you some code if you have several columns